tip-wlan/charts/opensync-gw-cloud/resources/config/certs/httpClientConfig.json

```diff
@@ -1,18 +1,21 @@
 {
-"maxConnectionsTotal":100,
-"maxConnectionsPerRoute":10,
-"truststoreType":"JKS",
-"truststoreProvider":"SUN",
-"truststoreFile":"file:/opt/tip-wlan/certs/truststore.jks",
-"truststorePass":"mypassword",
-"keystoreType":"JKS",
-"keystoreProvider":"SUN",
-"keystoreFile":"file:/opt/tip-wlan/certs/client_keystore.jks",
-"keystorePass":"mypassword",
-"keyAlias":"clientkeyalias",
-"credentialsList":[
-    {"host":"localhost","port":-1,"user":"user","password":"password"}
+    "maxConnectionsTotal": 100,
+    "maxConnectionsPerRoute": 10,
+    "truststoreType": "JKS",
+    "truststoreProvider": "SUN",
+    "truststoreFile": "file:/opt/tip-wlan/certs/truststore.jks",
+    "truststorePass": "mypassword",
```

Comment:
should not we parametrize the password here?

Reply:
nvm, just realized the formatting was changed.. needs to be fixed in a separate PR